src/client/extension.ts

```diff
@@ -58,7 +60,19 @@ export async function activate(context: IExtensionContext): Promise<IExtensionAp
         // We want to completely handle the error
         // before notifying VS Code.
         await handleError(ex, durations);
-        throw ex; // re-raise
+        traceError('Failed to active the Jupyter Extension', ex);
```

Comment:
I'm pro this, as I'm big on not busting python. But just to ask. Does this make an odd customer case if they just have Jupyter installed? Like it just fails for the user without visual notification? Wondering about either only doing this if Python is installed or popping up a message (I'm assuming that the re-raise is what would usually show the message) so that the user at least knows what happened.

Reply:
> uming that the re-raise is what would usually show the message) so that the user at least knows what happened.

I think its just to tell VS Code we failed.

`await handleError(ex, durations);`

this is where a message is displayed to the user today.
So if the extension fails to load they'll see a message (exiting code)